### DISCUSSION TITLE
开发杂谈

### DISCUSSION THREAD

**Discussion by BTMuli** [BTMuli/TeyvatGuide#74] [General]
[哔哩哔哩专栏](https://www.bilibili.com/read/cv28911185)

**Comment by findre** [comment]
你好，我也在开发一款基于Tauri的应用。

我的应用需要在用户完成其他平台的登陆操作后获取 cookie。但是我没有看到 tauri 提供过相关API。这就导致我的应用需要手动输入cookie才能完成与其他平台的交互。

你在开发这款应用的过程中有遇到过类似的问题吗？除了手动输入cookie有没有什么好的方案？
